src/protocol/protocol.c

```diff
@@ -405,7 +405,11 @@ int PROTOCOL_HasPowerAmp(int idx)
     } else {
 #endif        
     int m = get_module(idx);
-    if(m != TX_MODULE_LAST && Transmitter.module_poweramp & (1 << m))
+    if ((m != TX_MODULE_LAST && Transmitter.module_poweramp & (1 << m))
+#ifndef MODULAR
+        || (Model.protocol == PROTOCOL_PXX)
```

Comment:
why this is model is so special here?

Reply:
It's because we don't have the uart defined as a radio module, so it can't have a poweramp setting.  Adding code for a uart radio module didn't seem like good solution because the actual radios installed in the JR module bay can be different.  Checking for PXX protocol is a special case solution.  If we add support for other uart radio modules it will probably be best to look for a more general solution.